### DISCUSSION TITLE
Linux Support

### DISCUSSION THREAD

**Discussion by linsaftw** [leapmotion/UnityModules#1189] [Q&A] [answered]
We are working on a project with most of our workforce running on Linux OS.

When trying to run the example on a Linux machine (Lubuntu 21) we receive the following error: "DllNotFoundException: LeapC".

**Comment by Ultraleap-Dan** [comment, marked as answer]
Linux support is on our product roadmap, we are limited in what we can share at the moment but please keep an eye on our social news feeds for any annoucements that we make about this! :)

**Reply by linsaftw** [reply]
Yeah I understand but we are no "group of gamers with hype" we just want to work and we have deadlines to follow.

**Reply by Ultraleap-Dan** [reply]
We have an early release for our latest hardware nearly ready to go. Please email me at support@ultraleap.com with which camera hardware you are using and I'll be in touch!

**Reply by linsaftw** [reply]
I already quitted Linux because of the lack of support from developers.
